features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml

```diff
@@ -1430,6 +1430,24 @@
 				<SupportedByDefault />
 				<MappedLocalClaim>http://wso2.org/claims/url</MappedLocalClaim>
 			</Claim>
+			<Claim>
+				<ClaimURI>upn</ClaimURI>
+				<DisplayName>User Principal</DisplayName>
+				<AttributeID>upn</AttributeID>
```

Comment:
here AttributeID should be an available attribute in the default LDAP schema [1].
can't we map 'uid' as AttributeID here?

[1] https://github.com/wso2-extensions/identity-userstore-ldap/blob/master/features/org.wso2.carbon.ldap.server.server.feature/resources/conf/wso2Person.ldif

Reply:
@emswbandara Yes my mistake. It should map to `uid`. Thanks for pointing out. Fixed with 1c6ddc3ba697f347c87ab31fc9bae90508fc3644